streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/provider/TwitterFollowingProviderTask.java

```diff
@@ -141,7 +141,7 @@ else if( endpoint.equals("friends") )
             catch(Exception e) {
                 keepTrying += TwitterErrorHandler.handleTwitterError(client, e);
             }
-        } while (curser != 0 && keepTrying < 10);
+        } while (curser != 0 && keepTrying < 50);
```

Comment:
that would be a better approach for sure.

reversed this line and created STREAMS-330.